ui/test/resource/mock-stubs.ts

```diff
@@ -336,6 +336,11 @@ export class RichSkillServiceStub {
   ): Observable<PaginatedSkills> {
     return of(createMockPaginatedSkills())
   }
+
+  // noinspection JSUnusedGlobalSymbols,JSUnusedLocalSymbols
+  libraryExport(): Observable<string> {
+    return of(`x, y, z`)
```

Comment:
Does this do anything of value?  Did you intend to keep it?

Reply:
I will probably change it to something that resembles csv data. The other tests that return csv had this as the return hence why i tried it to test if it works for now

Reply:
@wgu-edwin, if this change isn't being used, let's remove it? I can't tell if it's providing data for a test or not.